octoprint_mrbeam/filemanager/file_storage.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+import os
+import time
+
+
+from octoprint.filemanager.storage import LocalFileStorage, StorageError
+from octoprint_mrbeam.mrb_logger import mrb_logger
+
+
+class MrBeamFileStorage(LocalFileStorage):
+    def __init__(self, basefolder, create=False):
+        self._logger_mrb = mrb_logger(
+            "octoprint.plugins.mrbeam.filemanager.mrbFileStorage"
+        )
+
+        LocalFileStorage.__init__(self, basefolder, create)
+
+    def remove_multiple_files(self, files):
+        metadata_to_remove = {}
+        f_count = 0
+        ts_start = time.time()
+
+        for path in files:
+            path, name = self.sanitize(path)
+
+            file_path = os.path.join(path, name)
+            if not os.path.exists(file_path):
+                continue
+            if not os.path.isfile(file_path):
+                self._logger_mrb.warn(
+                    "remove_multiple_files(): {name} in {path} is not a file -> skipping".format(
+                        name=name, path=path
+                    )
+                )
+
+            try:
+                os.remove(file_path)
+                f_count += 1
+            except Exception as e:
+                self._logger_mrb.error(
+                    "Could not delete {name} in {path}, cause: {cause}".format(
+                        name=name, path=path, cause=e
+                    )
+                )
+                continue
```

Comment:
Prefer `OSError` and `FileNotFoundError`:

```python
except OSError, FileNotFoundError:
    pass
```

Reply:
hmm... not sure if I agree. Anyway, this is copy-paste code from OP. Won't change